include/stack/stack.h

```diff
@@ -1,23 +1,20 @@
 /*
-    AUTHORS: Siddharth Mahendraker, Shimi Smith
+    AUTHORS: Siddharth Mahendraker, Shimi Smith, J. W. Sheridan
 */
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
-
-#define MAX_SIZE 50
-#define DATA_SIZE 8
+#define MAX_STACK_SIZE 50
+#define STACK_DATA_SIZE 8
 
 typedef struct {
 	uint8_t size;
     uint8_t index;
-    uint8_t content[MAX_SIZE][DATA_SIZE];
+    uint8_t content[MAX_STACK_SIZE][STACK_DATA_SIZE];
 } stack_t;
 
 void init_stack(stack_t*);
-uint8_t is_full(stack_t*);
-uint8_t is_empty(stack_t*);
-
+uint8_t stack_full(stack_t*);
+uint8_t stack_empty(stack_t*);
 uint8_t enstack(stack_t*, uint8_t*);
-uint8_t destack(stack_t*, uint8_t*);
+uint8_t destack(stack_t*, uint8_t*);
```

Comment:
Stupid annoying point; technically, all C files should end in a newline; you don't need to change this